frontend/src/views/ApplicantRegister.vue

```diff
@@ -1,85 +1,108 @@
-      >
-        <p class="summary" v-html="policySummary"></p>
-      </SummaryCheckField>
-      <TextField
-        v-model="name"
-        name="name"
-        type="text"
-        label="이름"
-        placeholder="이름을 입력해 주세요."
-        :rules="rules.name"
-        @valid="v => (this.validName = v)"
-        required
-      />
-      <TextField
-        v-model="phoneNumber"
-        name="phone-number"
-        type="text"
-        label="전화번호"
-        placeholder="연락 가능한 전화번호를 입력해 주세요."
-        :rules="rules.phoneNumber"
-        @valid="v => (this.validPhoneNumber = v)"
-        required
-      />
-      <TextField
-        v-model="email"
-        name="email"
-        type="email"
-        label="이메일"
-        placeholder="이메일 주소를 입력해 주세요."
-        :rules="rules.email"
-        @valid="v => (this.validEmail = v)"
-        required
-      />
-      <TextField
-        v-model="password"
-        name="password"
-        type="password"
-        label="비밀번호"
-        placeholder="비밀번호를 입력해 주세요."
-        :rules="rules.password"
-        @valid="validPasswordInputs"
-        required
-      />
-      <TextField
-        v-model="rePassword"
-        name="re-password"
-        type="password"
-        label="비밀번호 확인"
-        placeholder="비밀번호를 다시 한번 입력해 주세요."
-        :rules="[...rules.rePassword, v => v === password || '비밀번호가 일치하지 않습니다.']"
-        @valid="validPasswordInputs"
-        required
-      />
-      <BirthField v-model="birth" @valid="v => (this.validBirth = v)" required />
-      <GenderField v-model="gender" @valid="v => (this.validGender = v)" required />
-      <template v-slot:actions>
-        <Button cancel value="취소" />
-        <Button
-          type="submit"
-          :disabled="
-            !policyCheck ||
-              !validName ||
-              !validPhoneNumber ||
-              !validEmail ||
-              !validPassword ||
-              !validBirth ||
-              !validGender
-          "
-          value="다음"
-        />
-      </template>
-    </Form>
+    <ValidationObserver v-slot="{ handleSubmit, passed }">
+      <Form @submit.prevent="handleSubmit(submit)">
+        <h1>지원자 정보</h1>
+        <ValidationProvider rules="required" v-slot="{ errors }">
+          <SummaryCheckField
+            name="policy"
+            label="개인정보 수집 및 이용 동의"
+            v-model="policyCheck"
+            required
+          >
+            <p class="summary" v-html="policySummary"></p>
+          </SummaryCheckField>
+          <p class="rule-field">{{ errors[0] }}</p>
+        </ValidationProvider>
+
+        <ValidationProvider rules="name|required" :bails="false" v-slot="{ errors }">
+          <TextField
+            v-model="name"
+            name="name"
+            type="text"
+            label="이름"
+            placeholder="이름을 입력해 주세요."
+            required
+          />
+          <p v-for="(error, index) in errors" :key="index" class="rule-field">{{ error }}</p>
+        </ValidationProvider>
```

Comment:
```suggestion
        <ValidationProvider rules="name|required" v-slot="{ errors }">
          <TextField
            v-model="name"
            name="name"
            type="text"
            label="이름"
            placeholder="이름을 입력해 주세요."
            required
          />
          <p class="rule-field">{{ errors[0] }}</p>
        </ValidationProvider>
```

유효하지 않는 경우 메시지를 전부 보여주기 보단 첫번째로 유효성 검사 실패한 메시지만 보여주는 게 어떨까요?
다른 부분도 마찬가지입니다.